src/main/java/org/mitre/stix/DocumentUtilities.java

```diff
@@ -35,21 +35,31 @@
 
 /**
  * A collection of utility helper methods.
- * 
+ *
  * @author nemonik (Michael Joseph Walsh <github.com@nemonik.com>)
  */
 public class DocumentUtilities {
 
 	private static final String XML_SCHEMA_INSTANCE = "http://www.w3.org/2001/XMLSchema-instance";
 	private static final String XML_NAMESPACE = "http://www.w3.org/2000/xmlns/";
 
+	private static final JAXBContext STIX_CONTEXT = initDefaultContext();
+
+	private static JAXBContext initDefaultContext(){
+		try {
+			return JAXBContext.newInstance("org.mitre.stix.stix_1");
```

Comment:
I don't think these elements from other packages will be (often) directly converted with `toDocument`, I regard it as unlikely case. To me it is much more sensible to use it with STIXPackage. As for me, I think, that some more comments should be added to this method, preventing users from converting any possible element with it.